content/en/blog/how-to-use-fission-functions-with-postgresql-database/index.md

```diff
@@ -0,0 +1,357 @@
++++
+title = "How to use Fission functions with PostgreSQL database"
+date = "2021-12-27T12:28:34+05:30"
```

Comment:
Please change the date to today here. So that post appears in the feed for 2022.

Reply:
done